src/ixdat/spectra.py

```diff
@@ -710,15 +813,16 @@ def add_spectrum_series_to_measurement(measurement, spectrum_series, **kwargs):
 
 
 class SpectroMeasurement(Measurement):
+    child_attrs = ["spectrum_series_list"] + Measurement.child_attrs
     extra_column_attrs = {"spectro_measurements": {"spectrum_id"}}
 
-    def __init__(self, *args, spectrum_series=None, spec_id=None, **kwargs):
+    def __init__(self, *args, spectrum_series=None, spectrum_id=None, **kwargs):
```

Comment:
I don't understand. What should `SpectroMeasurement.join()` do?

Reply:
Aha, that is a consequence of inheriting from `Measurement` class. Since the `join()` method there raises `NotImplementedError` it means it is abstract, and it should be implemented in all classes that inherit from `Measurement`, otherwise the exception is raised. 
And when `SpectroMeasurement` is a measurement with spectrum series attached, I assumed it will be possible to join it with other `SpectroMeasurement`s?

Reply:
(note: we agreed that `join()` will be removed)